gdcdictionary/schemas/treatment.yaml

```diff
@@ -193,6 +315,14 @@ properties:
         $ref:
           - "_terms_enum.yaml#/other/common"
 
+  residual_disease:
```

Comment:
Should this and the other new properties have a `$ref`?

Reply:
They should, this should be resolved now.